script/brave_license_helper.py

```diff
@@ -155,6 +155,7 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
         os.path.join('brave', 'vendor', 'gn-project-generators'),
         os.path.join('brave', 'vendor', 'omaha', 'omaha', 'scons-out'),
         os.path.join('brave', 'third_party', 'libdmg-hfsplus'),
+        os.path.join('brave', 'tools', 'crates', 'vendor'),
```

Comment:
@bridiver is there a reason why these creates are not in `brave/third_party`? I'd move them while we're actively working in this area.

Reply:
Mainly to keep them separate from code used in the browser. Where would they go? brave/third_party/tools/crates or something